.circleci/config.yml

```diff
@@ -86,54 +86,49 @@ jobs:
       - store_test_results:
           path: ./results
 
-  test_7:
+  test_zulu_7:
     <<: *default_test_job
     environment:
       - TEST_TASK: testJava7
 
-  test_8:
+  test_zulu_8:
     <<: *default_test_job
     environment:
-      # We are building on Java8, this is our default JVM so no need to set more homes
-      - TEST_TASK: test jacocoTestReport jacocoTestCoverageVerification
+      # default JDK provider is ZULU, hence home set
+      - TEST_TASK: testJavaZULU8 jacocoTestReport jacocoTestCoverageVerification
 
-  test_latest8:
+  test_latest_8:
     <<: *default_test_job
     environment:
       # We are building on Java8, this is our default JVM so no need to set more homes
       - TEST_TASK: latestDepTest
 
-  test_ibm8:
-    <<: *default_test_job
-    environment:
-      - TEST_TASK: testJavaIBM8
-
-  test_zulu8:
+  test_zulu_9:
     <<: *default_test_job
     environment:
-      - TEST_TASK: testJavaZULU8
+      - TEST_TASK: testJava9
 
-  test_11:
+  test_zulu_10:
     <<: *default_test_job
     environment:
-      - TEST_TASK: testJava11
+      - TEST_TASK: testJava10
 
-  test_zulu11:
+  test_zulu_11:
     <<: *default_test_job
     environment:
       - TEST_TASK: testJavaZULU11
```

Comment:
Why is this different from others? `testJava10` but `testJavaZULU11`?

Reply:
It's confusing I admit but stems directly from how our gradle build creates test tasks. Names of the tasks are created out of Docker env variables and creator of the image (data dog employee) decided to name some ZULU JDKs with "zulu" in the name and some without (8 and 11 have zulu in name). To help avoid the confusion I named jobs accordingly (with zulu in the name added everywhere). To make it better we'd need to change the gradle build as well but I didn't want to modify too much. Hope that makes more sense now :)